### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by cbeach47**
Adds the `OS Slash` check for `CheckFiles`, and makes note to update `Config.VERSION` when releasing (which precipitated release instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,4 +138,17 @@ Uncomment everything to do with Kaltura (this needs to be fixed).
...
+1. Checkout `master`
+2. Run the tests (see `Building` section above)
+3. Update Config.VERSION to the release version of the Toolbox
+4. git tag the code with the release version
+5. Build the code with `./gradlew clean build`
+6. Push the changes and tag to master
+6. Manually create a release in github with the following:
+  1. blank.properties
+  2. log4j2.xml
+  3. openequella-toolbox-<version>.jar ( from `build/libs` )
```
Comment (ChristianMurphy) [inline README.md]: I'm not sure I follow this step, what needs to happen with these files?

File: README.md [hunk 2]
```diff
@@ -138,4 +138,17 @@ Uncomment everything to do with Kaltura (this needs to be fixed).
...
+
+Currently, it's a somewhat manual process:
+
+1. Checkout `master`
+2. Run the tests (see `Building` section above)
+3. Update Config.VERSION to the release version of the Toolbox
+4. git tag the code with the release version
+5. Build the code with `./gradlew clean build`
+6. Push the changes and tag to master
+6. Manually create a release in github with the following:
```
Comment (ChristianMurphy) [inline README.md]:
```suggestion
6. Push the changes and tag to master
7. Manually create a release in github with the following:
```

File: README.md (hunk 1 above)
Comment (cbeach47) [reply README.md]: I added some clarification - just needs to be uploaded at release artifacts

**Review by ChristianMurphy — APPROVED** [review]
Thanks @cbeach47 !

**Pull request merged**